### PR TITLE
Stories

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: styles/colors.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
 :root {
   --color-container: rgba(0, 0, 0, 5%);
+  --color-button: rgb(98, 185, 178);
```
Comment (JelenadlS) [inline styles/colors.css:3]: I am not sure, but i think it might be more handy to write the color name instead of container and button?

**Review by JelenadlS — CHANGES_REQUESTED** [review]
well done 👍🏼

File: index.html [hunk 2]
```diff
@@ -35,29 +35,43 @@ <h2>Question:</h2>
     </main>
     <nav class="app-grid__footer">
       <ul role="list" class="navigation">
-        <li>
+        <li class="navigation__button navigation__button--active">
```
Comment (JelenadlS) [inline index.html]: I am a little confused but on the other hand i did not start yet with the navigation. The active modification should apply to each button and then only when it is active right? dont you need to apply this on all buttons, so in `<ul>`?

**Review by JelenadlS — COMMENTED** [review]
I think it is a great work so far 😊

File: index.html (hunk 2 above)
Comment (ahohnsen) [reply index.html]: I actually just want to change the color of the button of the active page. So if you are on the index.html I want to have the home button in another color :)

File: styles/colors.css (hunk 1 above)
Comment (ahohnsen) [reply styles/colors.css:3]: Thank you. I thought about that but the one color is just like a shade and its easier for me to remember, when I name them in that way. But probably you are right and I should give them color names. Could be too confusing in a bigger project. Thanks :)

File: create.html [hunk 3]
```diff
@@ -7,54 +7,75 @@
...
     <link rel="stylesheet" href="./styles/index.css" />
   </head>
-  <body class="app-grid h-100vh">
+  <body class="app-grid">
     <h1 class="app-grid__header" id="create-heading">Create</h1>
-    <main class="app-grid__scrollarea m-20">
-      <form action="" aria-labelledby="create-heading">
+    <main class="app-grid__scrollarea">
+      <form class="form" action="" aria-labelledby="create-heading">
         <label for="question-field"
```
Comment (MichiBerger) [inline create.html:14]: really nice code and really good to read 👍 just with the label...if you wrap the label around the textarea, then a for is not necessary...but it´s not wrong and even maybe better if you want to change it later ;)

**Review by MichiBerger — APPROVED** [review]
really nice work ;)

File: create.html (hunk 3 above)
Comment (ahohnsen) [reply create.html:14]: Thank you so much! That's a good hint.

**Pull request merged**